internal/scan-manager/webapp/src/main/java/org/wso2/security/tools/scanmanager/webapp/config/ScanManagerWebappConfiguration.java

```diff
@@ -61,11 +62,11 @@ public static ScanManagerWebappConfiguration getInstance() {
     }
 
     public char[] getClientId() {
-        return clientId;
+        return Arrays.copyOf(clientId, 100);
```

Comment:
Why do we need new copy to have 100 elements? Shouldn't it be of same length as the original array?

Reply:
This will be fixed with the scan manager webapp PR